server/lib/src/constants/entries.rs

```diff
@@ -361,6 +363,7 @@ impl From<Attribute> for &'static str {
     fn from(val: Attribute) -> Self {
         match val {
             Attribute::Account => ATTR_ACCOUNT,
+            Attribute::DomainLdapAllowUnixPwBind => ATTR_DOMAIN_LDAP_ALLOW_UNIX_PW_BIND,
```

Comment:
Should be sorted alphabetically in the list

Reply:
Done